test/test_transforms_v2_refactored.py

```diff
@@ -187,7 +191,7 @@ def check_functional(functional, input, *args, check_scripted_smoke=True, **kwar
 
     assert isinstance(output, type(input))
 
-    if isinstance(input, tv_tensors.BoundingBoxes):
+    if isinstance(input, tv_tensors.BoundingBoxes) and functional is not F.convert_bounding_box_format:
```

Comment:
For `to_dtype` we added an extra flag to the single entrypoint check methods:

https://github.com/pytorch/vision/blob/1f94320d8db8d102214a7dc02c22fa65ee9ac58a/test/test_transforms_v2_refactored.py#L154-L155

https://github.com/pytorch/vision/blob/1f94320d8db8d102214a7dc02c22fa65ee9ac58a/test/test_transforms_v2_refactored.py#L1746

I'm ok doing the same here for the bounding box format.

However, since there will only ever be a single transform that needs this flag, I'm leaning towards hardcoding the exception inside the check function as done here.

LMK what you prefer. Depending on that, I'll either remove the dtype flag and hardcode the exception as well or add another flag for the bounding boxes format.

Reply:
as done here is fine